app/javascript/utils/ExcelExport.js

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
I would just call this file `Export.js` since it's not specific to excel

Thanks for breaking this out though, I think this was a good decision!

Reply:
Fixed in https://github.com/SaraAlert/SaraAlert/pull/805/commits/9e3e31175674140efbeb877b6bbbf711cb09768f